client-app/src/examples/contact/TileView.scss

```diff
@@ -0,0 +1,19 @@
+.contact-fb-tile {
+  border: var(--xh-border-solid);
+  margin: var(--xh-pad-px);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  &--selected {
+    border-color: var(--xh-orange);
+  }
+
+  .floating-name {
+    position: absolute;
+    font-weight: bold;
+    text-shadow: 2px 0 0 #000, 0 -2px 0 #000, 0 2px 0 #000, -2px 0 0 #000;
+    right: 20px;
+    bottom: 10px;
+  }
```

Comment:
These were looking weird when I ran locally:

<img width="402" alt="Screen Shot 2021-05-19 at 4 20 48 PM" src="https://user-images.githubusercontent.com/225806/118897537-5f2d4a00-b8bf-11eb-896d-1dd2b2c3da7b.png">

Here's a quick pass at that approach I mentioned using a semi-transparent background on the name div - could still be tuned a bit I am sure, but looks promising and could also hold a star to indicate / toggle fave status.

<img width="331" alt="Screen Shot 2021-05-19 at 4 27 41 PM" src="https://user-images.githubusercontent.com/225806/118897579-710eed00-b8bf-11eb-9dec-406d08d913cd.png">